Modules/core/Model/CoreInstall.php

```diff
@@ -633,6 +633,16 @@ public function upgrade_v3_v4() {
                 Configuration::getLogger()->debug('[stats] import clients, done!');
             }
         }
+
+        Configuration::getLogger()->debug('[qo_quotes] add column id_client');
+        $sql = "ALTER TABLE qo_quotes ADD COLUMN id_client INT(11) NOT NULL";
+        $this->runRequest($sql);
+        Configuration::getLogger()->debug('[qo_quotes] add column id_client, done!');
+
+        Configuration::getLogger()->debug('[qo_quotes] add column recipient_email');
+        $sql = "ALTER TABLE qo_quotes ADD COLUMN recipient_email VARCHAR(100) NOT NULL";
```

Comment:
not null? why as it is optional

Reply:
I was thinking about making it mandatory...